src/main/java/com/mraof/minestuck/client/gui/playerStats/AtheneumScreen.java

```diff
@@ -0,0 +1,93 @@
+package com.mraof.minestuck.client.gui.playerStats;
+
+import com.mojang.blaze3d.systems.RenderSystem;
+import com.mojang.blaze3d.vertex.PoseStack;
+import com.mraof.minestuck.MinestuckConfig;
+import com.mraof.minestuck.inventory.AtheneumMenu;
+import com.mraof.minestuck.network.AtheneumPacket;
+import com.mraof.minestuck.network.MSPacketHandler;
+import net.minecraft.client.renderer.GameRenderer;
+import net.minecraft.client.resources.sounds.SimpleSoundInstance;
+import net.minecraft.network.chat.Component;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.sounds.SoundEvents;
+import net.minecraft.world.entity.player.Inventory;
+
+import java.time.LocalDate;
+import java.time.Month;
+
+public class AtheneumScreen extends PlayerStatsContainerScreen<AtheneumMenu>
+{
+	public static final String TITLE = "minestuck.atheneum";
+	
+	private static final ResourceLocation guiBackground = new ResourceLocation("minestuck", "textures/gui/gui_inv_atheneum.png");
+	private static final ResourceLocation icons = new ResourceLocation("minestuck", "textures/gui/icons.png");
+	
+	private static final int upArrowY = 14, downArrowY = 48, arrowX = 151;
```

Comment:
My bad. I think it's like that because it was copy-pasted from EditmodeInventoryScreen and I just forgot to change it. I should probably go ahead and make EditmodeInventoryScreen's style consistent with everything else too, while I'm at it.